### DISCUSSION TITLE
Ajouter une case "Certification Ecocert" dans le diagnostic

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1251] [Idées]
Plusieurs utilisateurs m'ont fait remarquer qu'ils aimeraient bien que la mention de leur certification Ecocert apparaisse sur la fiche de leur cantine pour 2 raisons :
1. comme il s'agit d'une certification écologique très exigeante, elle mérite d'être mentionnée
2. comme pour obtenir cette certification, les chiffres d'appros d'une cantine doivent être authentifiés par un tiers extérieur, la mention de cette certification permettrait de distinguer les établissements dont l'exactitude des chiffres ne repose que sur la probité du déclarant et les établissements dont l'exactitude des chiffres est certifié par un organisme certifiant.

J'avoue, à titre personnel, avoir été convaincu par ce discours : je pense que c'est une très bonne idée!

**Comment by ghost** [comment]
Un autre utilisateur (E. D.) m'a fait part de ce besoin aujourd'hui.

**Reply by valeriemerle** [reply]
On pourrait leur proposer de faire cette remarque (qu'ils sont certifiés Ecocert) dans le champ libre qui leur est proposé quand ils publient dans "nos cantines". Cela aurait aussi l'avantage de pousser à publier?

**Reply by JenniferStephan** [reply]
je suis d'acc avec Val pour l'instant, cela reste une option à valoriser mais dans le champ libre. 
Après, c'est une question intéressante pour faire levier avec les Agores (qui délivre le label) et Restau'co; on pourrait négocier avec Christophe et Marie-Cecile; pour mettre en avant leurs adhérents, de rajouter leurs logos + le label
mais à mon avis, uniquement à garder sous le coude pr inciter les 2 assos à ns créer des utilisateurices ds leurs adhérents

**Reply by ghost** [reply]
👍
